apps/client/src/components/FormInput.tsx

```diff
@@ -0,0 +1,48 @@
+import TextField from '@mui/material/TextField';
+import React, { useState } from 'react';
+
+interface FormInputProps {
+  label: string;
+  type: string;
+  name: string;
+  value: string;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  icon?: string;
+  placeholder?: string;
+  required?: boolean;
+  helperText?: string;
+  error?: boolean;
+}
+
+const FormInput: React.FC<FormInputProps> = ({
```

Comment:
No reason to have this wrapper component for an MUI component, you're adding two divs that as David says in the commentary bellow make the DOM tree messy, please remove this component